easybuild/easyconfigs/o/OpenKIM-API/OpenKIM-API-2.0.1-intel-2018b.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenKIM-API'
+version = '2.0.1'
+
+homepage = 'https://openkim.org/'
+description = """Open Knowledgebase of Interatomic Models.
+
+OpenKIM is an API and a collection of interatomic models (potentials) for
+atomistic simulations.  It is a library that can be used by simulation programs
+to get access to the models in the OpenKIM database.
+
+This EasyBuild only installs the API, the models have to be installed
```

Comment:
You may want to update this text block to point to the openkim models easybuild

Reply:
I'll update the comment block.

As for the names, I think it is better to stay consistent with the naming conventions of EasyBuild, just as it is usually better for an API to be consistent with the conventions of the programming language rather than being consistent across languages.

@boegel Any opinion on this?

Reply:
@ellio167 @boegel 
I think I should reconsider - after all, there are lots of all-lowecase names in EasyBuild, and I seem to remember some semantic difference between OpenKIM and KIM (standard and implementation, perhaps).

I still really dislike the -v2 part of the name.  It is redundant, given the version number, and there is no need to keep v1 and v2 side-by-side as v1 is already on its way out.  In the same way, I regret not dumping the 3 in the ASAP3 name at the latest when I made it pip-installable (once upon a time, ASE and ASAP version 2 could coexist with versions 3, but the twos are long gone).  Now it is too late, another package has claimed the asap name on PyPI.

Python 2 and 3 have a need to be able to coexist, but even here EasyBuild has departed from the tradition to name the package Python3!

I suggest ``kim-api``  and ``kim-models`` (or is it ``openkim-models``?).

Reply:
It would be `kim-api` and `openkim-models`.

I totally understand your aversion to the "-v2".  I think my arguments for its necessity for side-by-side usage still stand.  And model comparison (even between various versions of the same model) is one of the goals of the KIM project.  However, your comments and my observations of how the various Simulator codes are adopting the transition (basically dropping v1 support and having only v2 support) are making me rethink the importance of having seamless side-by-side support.  

If we were to officially drop the "-v2" we would need to do it quickly.  I've initiated an internal discussion on this.  @schiotz, I'll keep you posted either here or via other means.

Reply:
@ellio167 OK, I'll defer to the collective wisdom of your group :-)

Reply:
@ellio167 Did you reach a conclusion on the name?

Reply:
@schiotz Thanks for the ping.  Yes, indeed.  We just finished our discussion and decided to make tje switch to a more conventional “kim-api” package name.  We’ll be implementing and releasing this quite soon.

Reply:
@schiotz OK, kim-api-2.0.2 is released and there is a new version of the models

https://s3.openkim.org/kim-api/kim-api-2.0.2.txz
https://s3.openkim.org/archives/collection/openkim-models-2019-03-29.txz

Reply:
@ellio167 @boegel 
I am making a new version of the openkim modules with the suggested module names (version 2.0.2, including a patch for the "refex" bug).  This made a new release of Asap necessary, too.

I am aware that it may never be merged due to the conflicting ASE versions.  Next week, I will make similar builds for the 2019a toolchain, which should be unproblematic.